plugins/content/local/store.go

```diff
@@ -646,6 +656,10 @@ func (s *store) ingestPaths(ref string) (string, string, string) {
 	return fp, rp, dp
 }
 
+func (s *store) ensureIngestRoot() error {
+	return os.MkdirAll(filepath.Join(s.root, "ingest"), 0777)
```

Comment:
This is different from `ingest/ref` that uses `0755`. Not sure if bug.

Reply:
Why 0777?

Reply:
Just copying the old code. Commented as I think it should be 0755, but unrelated to this specific issue.

Reply:
Looked up history, and it looks like these permissions come all the way from the initial implementation, and were just copied through during refactors https://github.com/containerd/containerd/commit/9008471a846baaf5f0c97e5ad43faaa2493853fb